jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -302,7 +303,9 @@ public static String getSkaffoldSyncMap(
     // since jib has already expanded out directories after processing everything, we just
     // ignore directories and provide only files to watch
     Set<Path> excludesExpanded = getAllFiles(excludes);
-    for (FileEntriesLayer layer : jibContainerBuilder.describeContainer().getFileEntriesLayers()) {
+    for (LayerObject layerObject : jibContainerBuilder.toContainerBuildPlan().getLayers()) {
+      Verify.verify(layerObject instanceof FileEntriesLayer);
```

Comment:
Presumably no user will ever hit this, but should we include an error message if they do?